frontend/packages/employee-frontend/src/api/hidden.ts

```diff
@@ -4,14 +4,6 @@
 
 import { client } from './client'
 
-export async function triggerFamilyBatch() {
-  await client.post('/family/batch-schedule')
-}
-
-export async function triggerVtjBatch() {
-  await client.post('/person/batch-refresh/vtj')
-}
-
 export async function triggerDvvBatch() {
-  await client.post('/person/batch-refresh/dvv')
+  await client.post('/scheduled/dvv/update')
```

Comment:
This will fail when the related evaka-infra PR is merged, but will be left here for both a placeholder for hidden buttons as well as occasional need to do additional dvv refreshes in an environment where it is allowed